.travis.yml

```diff
@@ -25,9 +25,13 @@ before_install:
   - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then ./.travis-install-gcc; fi
   - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then export PATH="$PATH:$HOME/gcc-arm-none-eabi-6_2-2016q4/bin"; fi
   - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then export PATH="$PATH:$HOME/uncrustify-uncrustify-0.65/build"; fi
+  - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then pip install --user mbed-cli; fi
+  - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then pip install --user colorama pyserial prettytable jinja2 intelhex junit_xml pyyaml requests mbed_ls mbed_host_tests mbed_greentea beautifulsoup4 fuzzywuzzy; fi
```

Comment:
Is this going to slow down our travis runs significantly? (@ppannuto)

Reply:
In practice the answer seems to be no...even if it did is it too much of an issue (as long as it's not egregious)?

Reply:
Yeah, as long as its not tens of minutes longer I'm happy with it. I just don't have a good feel for Travis, and know that in the past installing things took a lot of time.

Reply:
These travis runs seem to be taking ~11 minutes

Reply:
I turned on python/pip caching for travis in a commit here, which should alleviate any issues

Reply:
This really should have gone on #32...